src/azul/__init__.py

```diff
@@ -655,6 +658,15 @@ def portal_db_bucket(self) -> str:
     def portal_db_object_key(self) -> str:
         return f'azul/{self.deployment_stage}/portals/{self.dss_deployment_stage(self.dss_endpoint)}-db.json'
 
+    @property
+    def layer_bucket(self) -> str:
+        return self.versioned_bucket
+
+    @property
+    def layer_object_key(self) -> str:
+        requirements_hash = file_sha1(Path(self.project_root) / 'requirements.txt')
+        return f'azul/{self.deployment_stage}/layers/{requirements_hash}.zip'
```

Comment:
```suggestion
        return f'azul/lambda_layers/{requirements_hash}.zip'
```

Reply:
I was considering making this a `cachedproperty` but decided not to because there is no precedence of it in this file.

Reply:
In fact, it shouldn't be a property at all. It should be a function (method) of the hash and the has should not be computed here. It's a bad smell that `config` looks at a file that's only present at build time. If I were to call this method at runtime it would fail.